iina/Extensions.swift

```diff
@@ -343,10 +343,10 @@ extension String {
     self = self.substring(to: self.index(endIndex, offsetBy: -num))
   }
 
-  public func occurancesOf(_ str: String, inRange r: Range<Index>?) -> Int {
+  public func countOccurances(of str: String, inRange r: Range<Index>?) -> Int {
```

Comment:
I see you've improved the signature, but again, I'd prefer `countOccurances(of string: String, in range: Range<Index>?)`. There are a couple reasons:

1. Parameter names are "public-facing"; they should be reasonably "nice". I'd avoid abbreviations and truncations since they serve can serve as pseudo-documentation. In this case, where you're already using `range`, I'd prefer that you used shorter names *inside* the function rather than in the signature.
2. We can drop range from `inRange` since type information is strong and readily available. Besides, there's not a whole lot of things that can go after an `in:`.